mirage-solo5.opam

```diff
@@ -43,3 +43,6 @@ Currently this package also includes the C stubs used by the Solo5 `console`,
 The OCaml runtime and C runtime required to support it are provided separately
 by the [ocaml-freestanding](https://github.com/mirage/ocaml-freestanding) package.
 """
+conflicts: [
+  [ "ocaml-freestanding" {<= "0.7.0"} ]
```

Comment:
why is this needed?

Reply:
as discussed elsewhere, could you remove the conflict? It adds unnecessary pressure to the opam solver.